mailgun_pkg/package.yaml

```diff
@@ -0,0 +1,36 @@
+# Step meta data:
+name: mailgun
+version: 1.0
+runtime: go
+exec: main
+
+steps:
+  send_email@1.0:
+    description: Send a formatted email to an email address
+    inputs:
+      Domain:
+        required: true
+        description: The sender domain
+      SenderUsername:
+        required: true
+        description: The sender username which the message will be from
+      ApiKey:
+        required: true
+        description: Mailgun API Key
+      Subject:
+        required: true
+        description: Email subject
+      HtmlMessage:
+        required: false
+        description: A formatted html message which may be stylized
+      PlainTextMessage:
+        required: true
```

Comment:
Is this `required` or is it an either/or situation with HtmlMessage?

Reply:
If you don't include a plain text message with your html message it is more likely to be flagged as spam. To avoid that the plain text is always required and should closely match the html message (if an html message is included). It also ensures that emails can be viewed even if html is turned off.